vm/rust/Cargo.toml

```diff
@@ -8,8 +8,8 @@ edition = "2021"
 [dependencies]
 serde = "1.0.171"
 serde_json = { version = "1.0.96", features = ["raw_value"] }
-blockifier = "=0.5.0-rc.3"
-starknet_api = "=0.8.0"
+blockifier = "=0.6.0-rc.0"
```

Comment:
Should we bump up cairo-lang-starknet-classes to 2.6.0 [here](https://github.com/NethermindEth/juno/blob/kirugan/update-pebble-1.1.0/starknet/rust/Cargo.toml#L11C1-L11C28)? This would align it with the blockifier version see [here](https://github.com/starkware-libs/blockifier/blob/v0.6.0-rc.0/Cargo.toml#L26).

Reply:
Good point, I don't expect major changes but at least everything will be aligned together. Thanks